packages/salesforcedx-sobjects-faux-generator/scripts/regenerateMinsSObjects.ts

```diff
@@ -44,10 +44,13 @@ async function createConnection(aliasOrUsername: string) {
 async function generateLocalSobjectJSON(connection: Connection) {
   const sobjectNames = [
     'Account',
+    'AccountHistory',
```

Comment:
what source refers to the 3 sobjectNames?

Reply:
None that we own. This set of sobejcts is a "educated guess" of what the minimum set might be, with this major release adding three more.